native-audio/app/src/main/java/com/example/nativeaudio/NativeAudio.java

```diff
@@ -57,9 +59,18 @@ protected void onCreate(Bundle icicle) {
         assetManager = getAssets();
 
         // initialize native audio system
-
         createEngine();
-        createBufferQueueAudioPlayer();
+
+        int sampleRate = 0;
```

Comment:
Might be better to default to sensible sample rates and buffer sizes. E.g. 41000 and 1024